ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml

```
@@ -868,3 +868,168 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )test-pr-no-cluster,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

```suggestion
    always_run: false
```
no need to build the image for every push